src/core/modules/Nebula.cpp

```diff
@@ -962,12 +994,18 @@ void Nebula::drawHints(StelPainter& sPainter, float maxMagHints, StelCore *core)
 		case NebBLL:
 		case NebBLA:
 		case NebRGx:
-		case NebGxCl:			
-			if (finalSize > 35.f)
-				Nebula::texGalaxyLarge->bind();
-			else
-				Nebula::texGalaxy->bind();
-			break;
+		case NebGxCl:
+		{
+			// The rotation angle in renderEllipticMarker() is relative to screen. Make sure to compute correct angle from 90+orientationAngle.
```

Comment:
After moving this block here, you can delete it from below.

Reply:
Indeed.